Hackney.Shared.CautionaryAlerts/Factories/ResponseFactory.cs

```diff
@@ -78,11 +78,18 @@ public static CautionaryAlertResponse ToResponse(this CautionaryAlertListItem do
                 Description = domain.CautionOnSystem,
                 Reason = domain.Reason,
                 AssureReference = domain.AssureReference,
-                PersonId = Guid.Parse(domain.PersonId),
+                PersonId = CheckIfExists(domain.PersonId),
                 PersonName = domain.Name
             };
         }
 
+        private static Guid? CheckIfExists(string personId)
+        {
+            if (personId == null)
```

Comment:
The check and return could be a static helper method which uses TryParse for Guid:

```
        public static Guid? CheckIfGuid(string s)
        {
            Guid guid;
            if (TryParse(s, out guid))
                return guid;
            return null;
        }

```